src/porepy/examples/fracture_damage.py

```diff
@@ -407,7 +413,7 @@ def collect_data(self) -> DamageSaveData:
     "dilation_damage_decay": 0.5,
     "friction_coefficient": 0.1,  # Low friction to get slip \approx bc displacement
     "dilation_angle": 0.1,
-    # "shear_modulus": 1e6,  # Suppress shear displacement
+    "shear_modulus": 1e6,  # Suppress shear displacement in the matrix.
```

Comment:
Is the comment correct?

Reply:
Also, for my understanding: A shear modulus of 1e6 makes the rock sufficiently stiff to get close to the analytical solution also with momentum balance included?

Reply:
Yes